### PR TITLE
Upgrade to Mopidy 3

### PR DESCRIPTION
**Pull request by s-martin**
Implement #803 to support #796

### CODE REVIEW

**Comment by s-martin** [comment]
For existing users I think `sudo apt upgrade` Is necessary, but I’m a little reluctant to add that to the install script, so later automatic upgrade to ver 4 may take place.

**Comment by s-martin** [comment]
The previous installation of Mopidy based on Python packages probably needs to be removed.

**Comment by ZyanKLee** [comment]
@s-martin this could be circumvented by pinning the package version to `3.*`:

**/etc/apt/preferences.d/mopidy** :
```
Package: mopidy
Pin: version 3.*
Pin-Priority: 1001
```

File: scripts/installscripts/buster-install-mopidy3.sh [hunk 1]
```diff
@@ -492,11 +492,21 @@ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
 # Install required spotify packages
 if [ $SPOTinstall == "YES" ]
 then
+    # keep major verson 3 of mopidy
+    sudo echo "Package: mopidy\nPin: version 3.*\nPin-Priority: 1001" > /etc/apt/preferences.d/mopidy
```
[the remark this answers is not in the document]
Comment (ZyanKLee) [reply scripts/installscripts/buster-install-mopidy3.sh]: pipes do not inherit the privileges of the former command, but always those of the user running the whole command. Working around that is possible by using `tee` with `sudo`.

**Comment by ZyanKLee** [comment]
can you test it? I'm lacking the time to do so at the moment.

**Comment by s-martin** [comment]
I'll do some tests.

**Comment by s-martin** [comment]
I think, if some people have tried the script successfully, we could merge the changes back to buster-install-default.sh

**Comment by s-martin** [comment]
Removed unnecessary commands to install py packags, because already installed by `apt`.

Just tested it and I could open the web interface and it generally worked.
But I don't have a Spotify account, so I couldn't test, if music is played.

**Comment by s-martin** [comment]
From my point of vie it could be merged, because it's a separate script anyway, so other people could test it.

**Pull request merged**